src/app/providers/events.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+import {Subject} from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class EventsService {
+    private refreshKey = new Subject<any>();
+    private refreshWallets = new Subject<any>();
+
+    publishRefresh(data: any) {
+        this.refreshKey.next(data);
+    }
+
+    getRefresh(): Subject<any> {
```

Comment:
Why getRefresh retuen refreshKey? The name seems mismatch?

Reply:
changed name of function